mne/bem.py

```diff
@@ -1063,14 +1063,20 @@ def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
     run_subprocess(cmd, env=env, stdout=sys.stdout, stderr=sys.stderr)
 
     if op.isfile(T1_mgz):
-        # XXX : do this with python code
+        new_info = _extract_volume_info(T1_mgz)
+        if new_info is None:
+            warn('nibabel is required to replace the volume info. Volume info'
```

Comment:
Actually, this works with older nibabel too. So this earlier comment is not true. Only the functions related to reading writing ``.surf`` files uses the new nibabel/defaults to mne. This is reading an mgz file. So basically, if nibabel is not installed, it works the same way as in mne0.12 (omitting vol_info).